content/graph/LCA.h

```diff
@@ -3,35 +3,27 @@
  * Date: 2020-02-20
  * License: CC0
  * Source: Folklore
- * Status: Somewhat tested
+ * Status: stress-tested
  * Description: Data structure for computing lowest common ancestors in a tree
  * (with 0 as root). C should be an adjacency list of the tree, either directed
- * or undirected. Can also find the distance between two nodes.
- * Usage:
- *  LCA lca(undirGraph);
- *  lca.query(firstNode, secondNode);
- *  lca.distance(firstNode, secondNode);
+ * or undirected.
```

Comment:
I think it's worth adding a comment about formula for distance between 2 nodes. Would help beginner teams.

Reply:
Added as a commented-out function. Not 100% sure I agree with it being useful, but let's go with it, it's just one line after all.